src/app/utility/extensions/mat-dialog.extensions.ts

```diff
@@ -57,7 +57,7 @@ import {
 import { Observable } from 'rxjs/internal/Observable';
 import { filter, map, mergeMap } from 'rxjs/operators';
 
-declare module '@angular/material/dialog/dialog' {
+declare module '@angular/material/dialog' {
```

Comment:
This change produces a compilation error (at least in the current version of `mdm-ui` using Angular 12). This is because the `declare module` is trying to reference the exact JavaScript file where the `MatDialog` type is located, so `@angular/material/dialog/dialog` is correct - equates to the file `node_modules/@angular/material/dialog/dialog.d.ts`. Changing the module reference means all the extension methods cannot be bound.

Reply:
@pjmonks Good point - I've made that change in my bigger branch successfully, but maybe it needs making once angular has been updated.  I've reverted this one for now and hopefully it'll go through this time